server/src/plugins/google-storage-utils.ts

```diff
@@ -0,0 +1,114 @@
+
+import * as storage from '@google-cloud/storage';
+import getEnv from '../envs';
+
+function sleep(ms: number) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+export async function uploadFile(gcsFileName: string, mimetype:string, in_memory_file:any): Promise<string | null> {
+    const env = getEnv();
+    let finished = false;
+    const googleStorageClient = new storage.Storage();
+    const bucketName = env.googleStorageBucket;
+    const bucket = googleStorageClient.bucket(bucketName);
+    const file = bucket.file(gcsFileName);
+  
+    const blob = file.createWriteStream({
+      metadata: {
+        contentType: mimetype,
+      },
+      public: true
+    });
+
+    blob.on('error', (err: any) => {
+      console.log(err);
+      return null;
+    });
+  
+    blob.on('finish', () => {
+      finished = true;
+    });
+
+    blob.end(in_memory_file);
+
+    while(finished===false){
+      await sleep(200);
+    }
+
+    return `https://storage.googleapis.com/${bucket.name}/${gcsFileName}`
+
+    // stream.on('readable', () => {
+    //   let chunk;
+    //   let in_process = false;
+    //   while (null !== (chunk = stream.read())) {
+    //     in_process = true;
+    //     blob.write(chunk);
+    //   }
+    //   if(!in_process) {
+        
+    //   }
+    // })
+  }
+  
+  export async function listFiles() {
+    const env = getEnv();
+    const googleStorageClient = new storage.Storage();
+  
+    const [files] = await googleStorageClient.bucket(env.googleStorageBucket).getFiles();
+  
+    console.log('Files:');
+    files.forEach(file => {
+      getMetadata(env.googleStorageBucket, file.name);
+    });
+    // [END storage_list_files]
+  }
+  
+  export async function getMetadata(bucketName: string, filename: string) {
+    // [START storage_get_metadata]
+    // Imports the Google Cloud client library
+    const {Storage} = require('@google-cloud/storage');
```

Comment:
Why do we have this require in here? If we are importing the library at the top in [line 2](https://github.com/Xivis/poap/blob/ADMINS/event-endpoints-refactor/server/src/plugins/google-storage-utils.ts#L2)

My best guess is to change the import at the top and name it in PascalCase since it's the class. But if you do this you will have to change some lines.
```typescript
import * as Storage from '@google-cloud/storage';
```

Reply:
mi bad, it's copy paste from google examples